openslides/assignments/static/js/assignments/site.js

```diff
@@ -254,6 +280,26 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
         $scope.save = function (assignment) {
             Assignment.save(assignment).then(
                 function(success) {
+                    // check form element and set item type (AGENDA_ITEM = 1, HIDDEN_ITEM = 2)
+                    var type = assignment.showAsAgendaItem ? 1 : 2;
+                    // save only if agenda item type is modified
+                    if (assignment.agenda_item.type != type) {
+                        assignment.agenda_item.type = type;
+                        Agenda.save(assignment.agenda_item);
+                    }
+                    //save parent_id if modified. selection "-1" results in "null"
```

Comment:
add blank after `//` and write first word of a sentence in capitals and add a `.` at the end.